Objects/namespaceobject.c

```diff
@@ -219,7 +219,9 @@ namespace_replace(PyObject *self, PyObject *args, PyObject *kwargs)
 static PyMethodDef namespace_methods[] = {
     {"__reduce__", (PyCFunction)namespace_reduce, METH_NOARGS,
      namespace_reduce__doc__},
-    {"__replace__", _PyCFunction_CAST(namespace_replace), METH_VARARGS|METH_KEYWORDS, NULL},
+    {"__replace__", _PyCFunction_CAST(namespace_replace), METH_VARARGS|METH_KEYWORDS,
+     PyDoc_STR("__replace__($self, **changes)\n--\n\n"
+        "Return a copy of the namespace object with new values for the specified attributes.")},
```

Comment:
Is there a reason why this docstring uses the term attributes, but all the others in this PR use the term fields?

Reply:
Because SimpleNamespace is described as "a simple attribute-based namespace". code.replace() talk about fields (I copied the docstring for `__replace__()`). Also, the set of fields is limited for all other objects, you cannot set an arbitrary attribute.